doc/source/visualizing/plots.rst

```diff
@@ -930,54 +933,137 @@ Use any of the colormaps listed in the :ref:`colormaps` section.
    slc.set_cmap(("gas", "density"), "RdBu_r")
    slc.save()
 
-The :meth:`~yt.visualization.plot_window.AxisAlignedSlicePlot.set_log` function
-accepts a field name and a boolean.  If the boolean is ``True``, the colormap
-for the field will be log scaled.  If it is ``False`` the colormap will be
-linear.
+Colorbar norms
+::::::::::::::
+
+Slice plots and similar plot classes default to log norms when all values are
+strictly positive, and symlog otherwise. yt supports two different interfaces to
```

Comment:
It would, if that worked ! `LogNorm` isn't flexible in that regard, and only accepts strictly positive values, so if we have even a single value `<= 0`, `SymLogNorm` is the only option.

Reply:
OK cool.